packages/web/src/scss/components/Box/index.html

```diff
@@ -0,0 +1,64 @@
+{{#> web/layout/default title="Box" parentPageName="Components" }}
+
+<section class="UNSTABLE_Section">
+  <div class="Container">
+    <h2 class="docs-Heading">Default</h2>
+    <div class="docs-Stack docs-Stack--start">
+
+      <div>Content</div>
+
+    </div>
+  </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <div class="Container">
+    <h2 class="docs-Heading">With Custom Padding</h2>
+    <div class="docs-Stack docs-Stack--start">
+
+      <fieldset class="mb-0 border-0 mb-500">
+        <legend>For demo purposes the box is bordered</legend>
+      </fieldset>
+      <div class="border-basic border-100 p-800 border-solid">With custom padding</div>
+      <div class="border-basic border-100 px-800 py-400 border-solid">With custom vertical and horizontal padding</div>
+      <div class="border-basic border-100 pt-200 pb-800 pl-600 pr-1200 border-solid">With custom padding for each direction</div>
+      <div class="border-basic border-100 px-400 px-tablet-800 px-desktop-1200 py-200 py-tablet-400 py-desktop-600 border-solid">With responsive padding</div>
+    </div>
+  </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <div class="Container">
+    <h2 class="docs-Heading">With Border</h2>
+    <div class="docs-Stack docs-Stack--start">
+
+      <fieldset class="mb-0 border-0 mb-500">
+        <legend>For demo purposes the box has custom padding</legend>
+      </fieldset>
+      <div class="border-basic border-100 p-800 border-solid">Without radius</div>
+      <div class="border-basic rounded-300 border-100 p-800 border-solid">With custom radius</div>
+      <div class="border-basic rounded-full border-100 p-800 border-solid">With full radius</div>
+      <div class="border-basic rounded-300 border-200 p-800 border-solid">With thicker border</div>
+      <div class="border-focus rounded-300 border-100 p-800 border-solid">With focus color</div>
+    </div>
+  </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <div class="Container">
+    <h2 class="docs-Heading">With Background Color</h2>
+    <div class="docs-Stack docs-Stack--start">
+
+      <fieldset class="mb-0 border-0 mb-500">
+        <legend>For demo purposes the box is bordered</legend>
+      </fieldset>
+      <div class="bg-primary border-basic border-100 p-800 border-solid">Primary Background</div>
+      <div class="bg-secondary border-basic border-100 p-800 border-solid">Secondary Background</div>
+      <div class="bg-tertiary border-basic border-100 p-800 border-solid">Tertiary Background</div>
```

Comment:
nitpicking, can we group similar classes together? in the example it would mean move `p-800` outside of borders?

It would also be nice to sort it as in CSS (width-style-color).

 ```suggestion
      <div class="bg-tertiary border-100 border-solid border-basic p-800">Tertiary Background</div>
```